src/language/lang_es.php

```diff
@@ -12,7 +12,7 @@
   'Play' =>
     'Jugar',
   'Welcome to the Facebook Capture the Flag Competition. By clicking "Play," you will be entered into the official CTF challenge. Good luck in your conquest.' =>
-    'Bienvenido a la competición CTF de Facebook. Al hacer clic en "Jugar," entrarás en la competición CTF. ¡Mucha suerte!.',
+    'Bienvenido a la competencia CTF de Facebook. Al hacer clic en "Jugar," entrarás en la competencia CTF. ¡Mucha suerte!.',
```

Comment:
(I am spanish) Competición is rigth. Competencia means another thing.
Competición: People figthing to do something and win.
Competencia: Situation of companies engaged in a market offering or demanding the same product or service. (RAE)